modules/programs/neomutt.nix

```diff
@@ -223,9 +223,11 @@ let
 
       # Extra configuration
       ${account.neomutt.extraConfig}
-    '' + optionalString (account.signature.showSignature != "none") ''
```

Comment:
An alternative may be to do something like
``` nix
setOption "signature" (
  if account.signature.showSignature == "none" then
    null
  else
    "${pkgs.writeText "signature.txt" account.signature.text}"
)
```
although I don't know if it is particularly better.

Reply:
I think I'm going to keep it as-is. I don't think that is any better than what I have.